pages/research/department/[id].tsx

```diff
@@ -155,6 +175,81 @@ const Page = ({
             </Card>
           </div>
         </Section>
+        <Section>
+          <h2>Travels</h2>
+          <Paragraph>
+            Lorem ipsum, dolor sit amet consectetur adipisicing elit. Amet
+            molestiae, sequi animi est dolor nihil qui id, aperiam assumenda
+            suscipit officia, veniam tenetur veritatis saepe! Recusandae animi
+            incidunt fuga perferendis!
+          </Paragraph>
+          <TravelsOfDepartmentPrismMap
+            topology={getCountries()}
+            topologyObject={"ne_admin_0_countries"}
+            projection={geoBertin1953()}
+            width={10}
+            length={10}
+            extrudeGeometryOptions={{
+              depth: 0.01,
+              bevelSize: 0.005,
+              bevelThickness: 0.005,
+              bevelSegments: 12,
+            }}
+            btorsByCountryByDepartment={btorsByCountryByDepartment}
+            department={department}
+          />
+        </Section>
+        <Section>
+          <h2>Alumni</h2>
+          <div className="mb-5">
+            <label>
+              <br />
+              Select a level you want to filter for:
+              <select
+                className="ml-4"
+                name="level"
+                onChange={(event) => setLevel(event.target.value)}
+                placeholder={"none selected"}
+              >
+                <option value={""}>All levels</option>
+                {levels.map((d) => (
+                  <option value={d.level ?? ""} key={d.level}>
+                    {d.level}
+                  </option>
+                ))}
+              </select>
+            </label>
+          </div>
+        </Section>
+
+        <div style={{ padding: "0 1em", width: "42%", height: "100%" }}>
```

Comment:
Please avoid using the `style` prop and use the tailwind utility classes with the `className` prop instead. This would correspond to the following:

```suggestion
        <div className="px-5 w-50 h-100">
```

However, I doubt it is necessary to explicitly set the height to `100%`. Instead of the padding I would rather use a margin – you could make use of the `Section` component for that. And if you want to make all of the new maps smaller you could use a container div with the tailwind class `max-w-2xl` or something alike (instead of adjusting the width of all `div`s separately)

Reply:
Ok.